src/components/CreateArticle.jsx

```diff
@@ -1,10 +1,16 @@
 import React, { Component } from "react";
 import { connect } from "react-redux";
 import { TextArea, TextInput, Button, Form, Box, CheckBox } from "grommet";
+import { Attachment, Trash } from "grommet-icons";
 import axios from "axios";
-
+import ImageUploading from "react-images-uploading"
 class CreateArticle extends Component {
```

Comment:
I always add a line between 6 and 7 but I'm not sure what's conventional

Reply:
done